addons/background-blur/userscript.js

```diff
@@ -0,0 +1,20 @@
+export default async function ({ addon, console }) {
+  const overlay = document.createElement("div");
+  overlay.classList.add("sa-blur-overlay");
+  document.body.insertBefore(overlay, document.body.firstChild);
+
+  const observer = new MutationObserver((mutations) => {
+    mutations.forEach(() => {
+      const existingOverlay = document.querySelector(".ReactModal__Content");
+
+      const modalOverlayExists = document.querySelector(".ReactModal__Overlay");
+      overlay.classList.toggle("blur", modalOverlayExists);
+      existingOverlay?.classList.toggle("visible", modalOverlayExists);
+    });
+  });
+
+  observer.observe(document.body, {
+    childList: true,
+    subtree: true,
+  });
```

Comment:
I'm pretty sure running a querySelector each time there's any change on the body would hurt performance, specially inside the editor.

Reply:
Hence why I did not go with this method...they seemed quite adamant though and I didn't feel like arguing

Reply:
Im not sure if i ever considered that, but this addon isn't something I really care that much about anymore, if someone else wants to try it they are welcome to